<?xml version="1.0"?>

<launch>
    <node pkg="object_tracker" type="object_tracker" name="object_tracker" output="screen">
      <param name="project_objects" type="bool" value="true" />
      <param name="verfication_services" type="string" value="" />
      <param name="min_height" type="double" value="-1.0" />
      <param name="max_height" type="double" value="1.0" />
    </node>
</launch>

